ioctl/client.go

```diff
@@ -293,6 +295,39 @@ func (c *client) QueryAnalyser(reqData interface{}) (*http.Response, error) {
 	return resp, nil
 }
 
+func (c *client) Match(in string, matchType string) string {
```

Comment:
This is the print content about the specific command action hash, not the interactive process with user. I don't think it should be put here

Reply:
So, should I put this function back to `actionhash.go`?

Reply:
yes, it's not common.  just used in this command.
Pls add test and change pr state from draft to open. Thx.